src/services/findAllReferences.ts

```diff
@@ -920,12 +920,18 @@ namespace ts.FindAllReferences {
 
             /** @param allSearchSymbols set of additional symbols for use by `includes`. */
             createSearch(location: Node | undefined, symbol: Symbol, comingFrom: ImportExport | undefined, searchOptions: { text?: string, allSearchSymbols?: Symbol[] } = {}): Search {
+                // Symbols for private identifiers have a compound escaped name that includes the class symbol id
+                // This means that we need to take the original name of the symbol and search for it instead of the escaped name
+                function getSymbolName(s: Symbol) {

```

Comment:
I think this needs to be in utitilities and called `getEscapedNameIOfSymbol` instead to make sure we don't confused with `symbol.name`

Reply:
@sheetalkamat I checked again and found a utility called `symbolName` which does something similar, I will use that instead.